README.md

```diff
@@ -54,6 +54,12 @@
 ## Turorials & workshops
 
 - [Making shareable docs with Quarto](https://openscapes.github.io/quarto-website-tutorial/) - A tutorial to make website with Quarto.
+- [Workshop: From R Markdown to Quarto](https://rstudio-conf-2022.github.io/rmd-to-quarto/) - A workshop for those who want to take their R Markdown skills and expertise and apply them in Quarto, the next generation of R Markdown.
+- [Getting started with Quarto](https://rstudio-conf-2022.github.io/get-started-quarto/) - "Get started with Quarto" workshop materials for rstudio::conf 2022 
```

Comment:
Add "Workshop:" as from the previous item.

Reply:
Do you mean "remove"?

Reply:
`[Getting started with Quarto]` to `[Workshop: Getting started with Quarto]`

Reply:
Done ✅